### PR TITLE
Hosted Gallery call to action - fix layout

### PR DESCRIPTION
**Pull request by lps88**
## Does this affect other platforms - Amp, Apps, etc?
Nope, but needs to be implemented in AMP gallery too, in progress.

## Screenshots
Before
image
After
image

### CODE REVIEW

File: static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ define([
                     return fastdom.write(function () {
                         var i;
                         for (i = 0; i < placeholders.length; i++) {
-                            placeholders[i].innerHTML = json.html;
+                            placeholders[i].innerHTML = placeholders[i].innerHTML + json.html;
```
Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js]: I recommend using this instead:

```
placeholders[i].insertAdjacentHTML('beforeend', json.html)
```

this won't invalidate the whole content of the element and ultimately render faster.

File: static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ define([
                 })
                 .then(function () {
                     performanceLogging.moduleEnd(moduleName);
-                })
+                });
```
Comment (regiskuckaertz) [inline static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js]: thanks 😃

**Review by regiskuckaertz — APPROVED** [review]
Nice! 👍

File: static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js (hunk 1 above)
Comment (lps88) [inline static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js]: Couldn't remember how to do that, thanks!

**Pull request merged**